README.md

```diff
@@ -143,29 +194,42 @@ google_forms: [
 googleforms_googlesheet:
 ```
 
+- [ ] Make sure that `refresh-googleforms` is set to "yes".
+- [ ] Optionally, if you are saving data to Google, specify a googlesheet ID you'd like the GitHub data to be saved to. This will only be relevant if you've set `data_dest` to `google`.
+
 ### Setting up Slido
 
+This is assuming you have already exported Slido data to a google drive folder. Navigate to the Google Drive folder that has the Slidos you'd like to collect. Get the URL for this folder. It should look like this:
+
+```
+https://drive.google.com/drive/u/1/folders/<SOME_FOLDER_ID_HERE>
+```
+
+Put the folder id from this URL in your `drive_id:` category in the `_config_automation.yml` file.
 ```
 ###### Slido ######
 refresh-slido: yes
 drive_id: 1XWXHHyj32Uw_UyaUJrqp6S--hHnM0-7l
 slido_googlesheet:
 ```
 
+- [ ] Make sure that `refresh-slido` is set to "yes".
+- [ ] Optionally, if you are saving data to Google, specify a googlesheet ID you'd like the GitHub data to be saved to. This will only be relevant if you've set `data_dest` to `google`.
+
 ### Setting up YouTube
 
+[Follow the steps from the above section to authenticate Google](#setting-up-google-authentication) -- make sure that the account you have authenticated for has access to the YouTube you wish to collect.
+
+Go to a video you'd like to collect data from. The URL should look something like this:
+```
+https://www.youtube.com/watch?v=<YOUTUBE_VIDEO_ID_HERE>
+```
+Extract the Youtube Video ID from the URL and put it in the `video_ids` section of the `_config_automation.yml` file. Delete the example IDs we have put there as placeholders.
+Do this for each video you'd like to collect data from and separate the IDs with commas.
+
```

Comment:
add a comment about setting `youtube_googlesheet` if you want to store data on google?

Reply:
Done!